### DISCUSSION TITLE
how to get the boudnary points in data pde？

### DISCUSSION THREAD

**Discussion by liangaomng** [lululxvi/deepxde#1606] [Ideas]
Hello, I want to ask how to get the boundary and cal the boundary loss by extract from trainx？because we want to modify loss
        
        trianx=self.data.train_points()

**Comment by Fankunkun123** [comment]
Hello, sorry to bother you. Have you solved your problem?
 I have a question for you:

>  If you have extracted the data point x, how did you modify the loss function or add the data loss to the total loss?

Thank you.

**Comment by lululxvi** [comment]
@liangaomng Check the docstring and comments of the code, and you will understand.
